### ISSUE TITLE
Issue: autoscale print targetrequests value instead of the pointer address

### ISSUE THREAD

**Issue by benz9527** [substratusai/kubeai#231]
![image](https://github.com/user-attachments/assets/329e4935-a94f-4c63-8d97-e822de728a6e)

![image](https://github.com/user-attachments/assets/641292b8-a38b-438c-93c2-772174237c91)

**Comment by nstogner** [comment]
Good catch! If you would like to be added as a contributor to the project feel free to submit a PR to tackle this (`m.Spec.TargetRequests` should never be nil in this case so it should just be fixed by de-referencing: `*m.Spec.TargetRequests`).

Otherwise, we will fix soon.

**Comment by benz9527** [comment]
I glad to make a PR.

**Comment by benz9527** [comment]
I try to push a new fix branch which is created from main, but I found that I haven't permission to create a remote branch. Haha.

**Comment by samos123** [comment]
You can create a fork and create a branch in your fork. Afterwards you can submit a PR.

**Comment by benz9527** [comment]
> You can create a fork and create a branch in your fork. Afterwards you can submit a PR.

Oh, I forget that github opensource project need to  be forked and then I can PR.
